packages/lesswrong/lib/abTestImpl.ts

```diff
@@ -60,7 +61,7 @@ type ABTestGroup = {
 }
 
 type ABKeyInfo = {
-  clientId: string
+  clientId?: string
```

Comment:
`clientId` can now be undefined on the server, in the case where a new user's first request is to a cache friendly page (we can't add a "set-cookie" to the response, so it is generated as soon as the client initialises instead)